### ISSUE TITLE
Panel 6 and lines

### ISSUE THREAD

**Issue by gdcumming** [gfmoore/esci-correlation#5]
0.0.7

I've spent a fair bit of time with the stats books and digging into the two ESCI workbooks, trying to get a good grasp once again. I think this is the simplest way to think of it, and calculate stuff:

**Regression of Y against X.** Let's say Y is in km and X in hours, then the slope is a speed in km/h.
Slope is bY.X = **r * SY/SX**
...where SY is SD of Y and SX is SD of X.
The regression line of Y against X has this slope and goes through the point (MX, MY) where they are the means. That's the point at which the cross through the means is drawn. (In fact, all three lines I mention go through this point.)

**Regression of X against Y.**  The books say that slope is bX.Y = r * SX/SY, which has units h/km
That's actually the slope for a graph with the axes transposed.
To plot this line on the familiar graph with Y on the vertical axis, X on the horizontal, the slope of the reg of X against Y is **(1 / r) * SY/SX**, in units km/h.

**In ITNS, see pp346-348** for a bit on reg of X against Y, without formulas. The explanation there of the Y against X line refers to it being horizontal then rotated anti-clockwise an amount determined by r. Small positive r and the line is not far from horizontal, large positive r and it is rotated up towards the r=1 line.

For reg of X against Y, the line is rotated clockwise from vertical by an amount determined by r. Small positive r and it is not far from vertical, large positive r and it approaches the r = 1 line.

Also noted is that reg of Y against X minimises the sum of squares of vertical deviations of points from the line, whereas the reg of X against Y minimises the sum of squares of horizontal deviations--when we are using the familiar graph, Y on the vertical, X on horizontal axis.

(Pardon me if most if not all of this is very familiar to you!)

**Correlation line, or r = 1 line.** (I haven't been able to find a single accepted term for this.) This line has slope that is the geometric mean of the slopes of the two regression lines. In other words, slope = **SY / SX**. (On the familiar graph.) This is the line towards which the two regression lines rotate, from opposite directions, as r approaches 1. When r = 1, all three lines coincide.

In 0.0.7 I've tried checking, for a few data sets, that the values reported for means, SD values, r, and slopes all tie together in accord with the above equations. In general I've found that all do tie together, except for the slope of the correlation line--which often seems considerably off. (The closeness of agreement within all the other values is limited I think by having values only to 2 d.p.)

In short, the three lines all go through the means, and have the slopes in bold above.

Your little test data set: ESCI calculates reg of Y against X is: Y = 0 + 0.8657 * X

In 'See r' the data set starts off as a random sample, then is tweaked until r is what we want. So it isn't a random sample from a particular population with correlation rho. So it doesn't really make sense to think of a CI on the slope, or on the regression line. So that shouldn't appear in Panel 6. (We'll keep such things for 'correlation'.)

I will send an 'opened up' version of ESCI intro. In fact I'll make that another issue--perhaps useful for testing.

**Comment by gfmoore** [comment]
Thinking I've got this now. I'm intrigued though why the eigenvectors didn't line up with the correlation line. I will experiment further with this as I find it very interesting, but for now I've removed it and will concentrate on this r iteration thing.

**Comment by gfmoore** [comment]
I still need to properly test though. I might create a little data import function - but later this week (I already have code, I just need to incorporate it.)

**Comment by gfmoore** [comment]
Made an adjustment to cater for negative r. in 0.0.8

**Comment by gdcumming** [comment]
0.0.8

Again I'm mainly eyeballing to test values, but it looks good.

Just two things:
1. I think we should have the 'X against Y' slope value match what we see in the scatterplot, so should be the reciprocal of what's currently reported. (I may have been vague or ambiguous on that, in the above, sorry.)

2. I like the 3 lines having three different colours. Let's have the labels in Panel 6 match the line colours. So 'Y against X' in blue, as now. 'X against Y' in red. 'Correlation line' in black. Let's leave 'Slope' and the values in blue and red as now. (It's the lines themselves that appear in colour.)

I note also that my description above talked about correlation line slope as the sqrt of the other slopes. But that needs a minus sign when r<0. I guess that's what you fixed. Goodie.

**Comment by gfmoore** [comment]
Done

**Comment by gdcumming** [comment]
0.0.9

Great, closing
